Example/Pods/Pods.xcodeproj/project.pbxproj

```diff
@@ -555,6 +558,7 @@
 				0211271CB4BD71A42BDE5CD492397A3C /* Chargebee.swift in Sources */,
 				D3BF6D273DD0446DBAC6E9E14035B96D /* Chargebee-dummy.m in Sources */,
 				52B6734620FC23310AB038FDC2E20417 /* StripeTokenizer.swift in Sources */,
+				FB23D7A92991447600BC402D /* CBCustomer.swift in Sources */,
```

Comment:
@cb-gadagoju can we remove this local changes?

Reply:
@cb-amutha  when we add a swift file to project it will generate it i dont think we can should remove it.

Reply:
Got it